caddy/caddymain/run.go

```diff
@@ -409,13 +415,61 @@ func initTelemetry() error {
 	return nil
 }
 
+// LoadEnvFromFile loads additional envs if file provided and exists
+//Envs in file should be in KEY=VALUE format
+func LoadEnvFromFile(envFile string) error {
+	if envFile == "" {
+		return nil
+	}
+
+	file, err := os.Open(envFile)
```

Comment:
Add a `defer file.Close()` here to close the file after reading.

Reply:
Sorry, add it after the error check below instead.

Reply:
Ok, sorry, i missed it:)